src/redux/books/book.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable arrow-body-style */
+const InitialState = [];
+
+export const addBook = (id, author, title) => {
+  return {
+    type: 'ADDED_BOOK',
+    payload: { id, author, title },
+  };
+};
```

Comment:
- [ ] Kindly make sure that the type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux). The type should be called as follows:
```javascript
type: '<app name>/<reducer name>/<action type>',
```
which, can be translated to the following:
```javascript
type: 'Bookstore/books/ADDED_BOOK',
```

Reply:
@ShadyShawkat Thank you very much for your review. I left you a message on slack